chain/cmd/bandoracled/main.go

```diff
@@ -97,6 +104,16 @@ $ bandoracled --node tcp://localhost:26657 --priv-key 06be35b56b048c5a6810a47e2e
 	viper.BindPFlag(flags.FlagNode, cmd.Flags().Lookup(flags.FlagNode))
 	cmd.Flags().IntP(flagMaxQueryDuration, "d", 60, "Max duration to query data")
 	viper.BindPFlag(flagMaxQueryDuration, cmd.Flags().Lookup(flagMaxQueryDuration))
+	cmd.Flags().String(flags.FlagFees, "", "Fees to pay along with transaction; eg: 10uband")
+	viper.BindPFlag(flags.FlagFees, cmd.Flags().Lookup(flags.FlagFees))
+	cmd.Flags().String(flags.FlagGasPrices, "", "Gas prices to determine the transaction fee (e.g. 10uband)")
```

Comment:
Why do we accept all three values - fee, gas, gasPrice? Is it possible that fee != gas * gasPrice?

Reply:
It cannot set fee and gasPrice together.